### ISSUE TITLE
[BUG] Manual Build of v0.9.4 fails with Error on Mobian/PinePhone

### ISSUE THREAD

**Issue by nuehm-arno** [nanu-c/axolotl#331]
### Description

Hello there,

`npm run build` fails with an error when trying to manually build Axolotl 0.9.4 on Mobian/PinePhone (see attached log file).

Thanks in advance - and thanks a lot for Axolotl! v0.9.0.1 is running on my PinePhone with some tweaks and I like it very much.

### Steps to Reproduce

1. Follow the installation guide in the [Mobian wiki](https://wiki.mobian-project.org/doku.php?id=axolotl)
2. `npm run build` fails

**Expected behavior:**
`npm run build` successfully completes

**Actual behavior:**
Output log file

> 29 error code 1
30 error path /home/mobian/go/src/github.com/nanu-c/axolotl/axolotl-web
31 error command failed
32 error command sh -c vue-cli-service build
33 verbose exit 1

#### Versions
v0.9.4

#### Device
PinePhone CE 3Gb with Mobian (with latest updates on FDE-image 20201215)

#### Link to Debug Log
See attached log file.
[Axolotl_2021-01-25T22_59_56_033Z-debug.log](https://github.com/nanu-c/axolotl/files/5875049/Axolotl_2021-01-25T22_59_56_033Z-debug.log)

**Comment by nanu-c** [comment]
Hey, can you try to run `npm install` in /home/mobian/go/src/github.com/nanu-c/axolotl/axolotl-web and then try it again?

**Comment by nuehm-arno** [comment]
Thanks for the fast answer. Sadly, it didn't help. Same error.
But I did a second `npm rebuild node-sass`, because I found an error message scrolling through the terminal logs about node-sass. It took quite more time to rebuild node-sass this time and afterwards, `npm run build` went fine.

I'm now on v0.9.4. Yay.


P.S.: `axolotl -e qt` is still not working because of the aforementioned errors [here](https://github.com/nanu-c/axolotl/issues/291#issuecomment-748978276)
